### PR TITLE
Feature/reservations duck

### PR DESCRIPTION
**Pull request by AdedayoOpeyemi**
### Added the following

- [x] Cancel reservation action 
- [x] Add reservation action
- [x] Get user reservation action

### CODE REVIEW

**Comment by RokoVarano** [comment]
There are linter errors in your pull request. There are also several incomplete code errors (unused variables, empty statements). Please fix them.

**Review by RokoVarano — DISMISSED** [review]
Please solve the linter errors so I can check functionality. Code seems to be incomplete.

File: src/redux/reservation/reservation_duck.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+};
+
+const reservations = (state = initialState, action) => {
+  switch (action.type) {
+    case POST_RESERVATION:
+      return state;
+    case GET_RESERVATION:
+      return state;
+    case LOAD_RESERVATION:
+      return action.payload;
```
Comment (notlfish) [inline src/redux/reservation/reservation_duck.js:69]: Is this getting an object with the same shape as the initial state, say
```javascript
{
  reservations: [arrayWithReservations]
}
```
Please, make sure that it does, as changing the shape of the store objects lends to hard-to-find bugs.

**Review by notlfish — CHANGES_REQUESTED** [review]
# Changes requested
### Thank you for correcting the linter errors!

There's one more change before merging, please see the comments.

**Review by notlfish — APPROVED** [review]

**Pull request merged**